test/test-utils.sh

```diff
@@ -343,7 +343,16 @@ function make_random_string() {
     else
         END_POS=8
     fi
-    RANDOM_STR=`cat /dev/urandom | base64 | sed 's#[/|+]##g' | head -1 | cut -c 1-${END_POS}`
+
+    # [NOTE]
+    # The number of bytes to be read is "((($ 1) / 2 + 1) / 4 + 1) * 4". 
+    # Since it is displayed in units of 4 bytes with the od command,
+    # it will be filled with 0 if it is not a 4-byte boundary.
+    # To receive the result in hexadecimal, you need at least half the
+    # number of bytes required.
+    #
+    TMP_LOAD_BYTES=$((((${END_POS}/2 + 1)/4 + 1)*4))
+    RANDOM_STR=$(od -An -N${TMP_LOAD_BYTES} -tx /dev/urandom | tr -d [:space:] | cut -c 1-${END_POS})
```

Comment:
Linux supports `od --width` but unfortunately macOS does not.

Reply:
This part is undone.